mlir/lib/ExecutionEngine/CMakeLists.txt

```diff
@@ -100,9 +101,9 @@ mlir_target_link_libraries(MLIRExecutionEngine PUBLIC
   MLIRLLVMToLLVMIRTranslation
   MLIROpenMPToLLVMIRTranslation
   MLIRTargetLLVMIRExport
-  )
+)
 
-if(LLVM_BUILD_LLVM_DYLIB AND NOT (WIN32 OR MINGW OR CYGWIN)) # Does not build on windows currently, see #106859
+if(LLVM_BUILD_LLVM_DYLIB AND NOT(WIN32 OR MINGW OR CYGWIN)) # Does not build on windows currently, see #106859
```

Comment:
Please avoid spurious whitespace changes. These increase greatly the amount of code to review when they should not have changed.

Reply:
Hi Renato (@rengolin ),

I apologize for this, I had a CMake formatter in my vscode which did auto-formatting :(.
Should I remove the formatted version?

Reply:
At least the parts that do not change otherwise. I'd recommend you disable auto-format in CMake files and only run on the lines you change and surrounding code. (Most editors can select code and run the formatter on the selection).

Reply:
Fixed now. Thanks, @rengolin :).